multiqc/modules/peddy/peddy.py

```diff
@@ -148,30 +162,64 @@ def peddy_general_stats_table(self):
             'title': 'Ancestry',
             'description': 'Ancestry Prediction',
         }
+        headers['ancestry-prob'] = {
+            'title': 'P(Ancestry)',
+            'description': 'Probability predicted ancestry is correct.'
+        }
         headers['sex_het_ratio'] = {
             'title': 'Sex / Het Ratio',
         }
         headers['error_sex_check'] = {
             'title': 'Correct Sex',
             'description': 'Displays False if error in sample sex prediction',
         }
+        headers['predicted_sex'] = {
+            'title': 'Sex',
+            'description': 'Predicted sex'
+        }
         self.general_stats_addcols(self.peddy_data, headers)
 
     def peddy_pca_plot(self):
-
+        ancestry_colors = {
+            'SAS': 'rgb(68,1,81,1)',
+            'EAS': 'rgb(59,81,139,1)',
+            'AMR': 'rgb(33,144,141,1)',
+            'AFR': 'rgb(92,200,99,1)',
+            'EUR': 'rgb(253,231,37,1)'
+        }
+        background_ancestry_colors = {
+            'SAS': 'rgb(68,1,81,0.1)',
+            'EAS': 'rgb(59,81,139,0.1)',
+            'AMR': 'rgb(33,144,141,0.1)',
+            'AFR': 'rgb(92,200,99,0.1)',
+            'EUR': 'rgb(253,231,37,0.1)'
+        }
+        default_color = '#000000'
+        default_background_color = 'rgb(211,211,211,0.05)'
         data = dict()
         for s_name, d in self.peddy_data.items():
             if 'PC1_het_check' in d and 'PC2_het_check' in d:
                 data[s_name] = {
                     'x': d['PC1_het_check'],
                     'y': d['PC2_het_check'],
+                    'color': ancestry_colors.get(d['ancestry-prediction'], default_color)
                 }
+            if s_name == "background_pca":
```

Comment:
Instead of doing this inside the loop, could you move it out and do it first? That way the sample points should always be on top, so that the hover tooltips work (in my test they were partially hidden):
![image](https://user-images.githubusercontent.com/465550/38955105-e8ca27ae-4353-11e8-8cfd-389aeeb19b0a.png)

![image](https://user-images.githubusercontent.com/465550/38955119-f4f8eace-4353-11e8-8929-6ca2412e5952.png)

Reply:
(it'll need to be an `OrderedDict`, but I think it should work...)

Reply:
Oh, that's a good idea, I will do that as a stopgap. I was hoping for the future we could like turn off the mouseover for those background points, make them the right ancestry colors and put borders around the real points if, point-specific features get implemented. ;)